quint/io-cli-tests.md

```diff
@@ -596,6 +596,32 @@ rm out-itf-example.itf.json
 ]
 ```
 
+### Test outputs ITF
+
+TODO: output states after fix: https://github.com/informalsystems/quint/issues/288
+
+<!-- !test in test itf -->
+```
+quint test --out-itf=coin_ \
```

Comment:
```suggestion
quint test --out-itf=coin_ \
```

Maybe `--output-prefix` (I assume we would also use this prefix for any other additional file output)?
And drop the `_` from the parameter and add it programmatically instead?

IMO that would also be more uniform with our treatment in Apalache.